src/commands/aksOpenKubectlPanel/openKubectlPanel.ts

```diff
@@ -0,0 +1,73 @@
+import * as vscode from "vscode";
```

Comment:
can you rename file to aksOpenKubectlPanel.ts please?

Reply:
Yes, updated now